### PR TITLE
MOAI-219 Early-Ed Validation Rules bbullard

### PR DESCRIPTION
**Pull request by bbullard-doubleline**

### CODE REVIEW

File: ValidationWeb/Content/Rules/EarlyEdSchoolRuleSet.rules [hunk 1]
```diff
@@ -0,0 +1,15 @@
+ruleset EarlyEdSchoolRuleSet
+
+rule 52.02.0001
+	when {SchoolCalendar}.[KindergartenScheduleCode] is in ['P','Q','R'] then
+	require that {SchoolCalendar}.[GradeLevel] is in [
```
Comment (steve-dlp) [inline ValidationWeb/Content/Rules/EarlyEdSchoolRuleSet.rules]: Replace SA-SJ and FA-FJ with EE

File: ValidationWeb/Content/Rules/StudentDemographic.rules [hunk 2]
```diff
@@ -319,6 +319,11 @@ rule 10.10.6600
 	else '202. The student\'s last name should have more than 1 character.'
 	
 rule 10.10.6601
-	when {StudentEnrollment}.[StudentGradeLevel] is not in ['PS','EC','PA','PB','PC','PD','PE','PF','PG','PH','PI','PJ','RA','RB','RC','RD','RE','RF','RG','RH','RI','RJ'] then
+	when {StudentEnrollment}.[StudentGradeLevel] is not in [
```
Comment (steve-dlp) [inline ValidationWeb/Content/Rules/StudentDemographic.rules]: Undo the change -  reverting the original rule back to what it was and add a copy of this to the early ed student ruleset and in that copy remove all gradelevels except the new 'EE" gradelevel.

File: ValidationWeb/Content/Rules/StudentEnrollment.rules [hunk 3]
```diff
@@ -27,12 +27,16 @@ rule 10.10.6214
 
 rule 10.10.6215
 	when {StudentEnrollment}.[DistrictType] is in ['1','2','3']
-	and {StudentEnrollment}.[StudentGradeLevel] is not in ['HK','EC','PS'] 
+	and {StudentEnrollment}.[StudentGradeLevel] is not in [
```
Comment (steve-dlp) [inline ValidationWeb/Content/Rules/StudentEnrollment.rules]: Undo the changes  here - reverting the original rule back to what it was and add a copy of this to the early ed student ruleset and in that copy remove all gradelevels except the new 'EE" gradelevel.

File: ValidationWeb/Content/Rules/Collections.rules [hunk 4]
```diff
@@ -21,4 +21,7 @@ collection McccValidationCollection includes
   
 collection ExamReimbursementRequestCollection includes
   ExamReimbursementRequestRuleSet
-
+  
+collection EarlyEdValidationCollection includes
```
Comment (steve-dlp) [inline ValidationWeb/Content/Rules/Collections.rules]: Change collection name to EarlyEdEnrollmentValidation

File: ValidationWeb/Content/Rules/EarlyEdEnrollmentStudentRuleSet.rules [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+	when {StudentEnrollment}.[StatusEnd] is in ['67','68','69','70'] then
+	require that {StudentEnrollment}.[StudentGradeLevel] = 'EE'
+	else 'Students with enrollment Exit Withdraw Type of 67-Student withdrawn by parent, expected back next section, 68-Student withdrawn by parent, expected back next year, 69-Student withdrawn by parent, moved to another district prenatal to prekindergarten program, or 70_Lack of transportation must be for grade level EE (Early Education).'
+	
+rule 52.01.0003
+	when {StudentEnrollment}.[StatusEnd] = '71' then
+	require that {StudentEnrollment}.[StudentGradeLevel] = 'EE'
+	else 'Students with an enrollment exit withdraw type 71-Short term course ended (ECFE only) must be for grade level EE (Early Education).'
+	
+rule 52.10.6601
```
Comment (bbullard-doubleline) [inline ValidationWeb/Content/Rules/EarlyEdEnrollmentStudentRuleSet.rules]: @steve-dlp rules from StudentEnrollment (with EE added) start here
